tools/cloud-build/daily-tests/tests/spack-gromacs.yml

```diff
@@ -15,14 +15,14 @@
 ---
 
 test_name: hpc-slurm-gromacs
-deployment_name: "spack-gromacs-{{ build }}"
+deployment_name: "spackgroma"
```

Comment:
If you don't have the build id in here then only 1 test can run in parallel. Please try to include build id in name.

Reply:
Included build name.